lib/krb5/pac.c

```diff
@@ -433,8 +433,9 @@ krb5_pac_add_buffer(krb5_context context, krb5_pac p,
      * copy in new data part
      */
 
-    memcpy((unsigned char *)p->data.data + offset,
-	   data->data, data->length);
+    if (data->length != 0)
```

Comment:
There is no need to check for non-zero data length since the data pointer is not permitted to be NULL.